PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -210,8 +210,22 @@ function New-Report {
 
     # Handle AAD-specific reporting
     if ($BaselineName -eq "aad") {
+        
+        # Load the CSV file
+        $csvData = Import-Csv -Path "Product Names for Licensing.csv"
+        
         $LicenseInfoArray = $SettingsExport.license_information | ForEach-Object {
+
+            $SkuID = $_.SkuId
+            # Find the corresponding product name
+            $matchingRow = $csvData | Where-Object { $_.GUID -eq $SkuID } | Select-Object -First 1
+            $productName = ""
+            if ($matchingRow) {
+                $productName = $matchingRow.'Product_Display_Name'
+            }
+            # Create a custom object with relevant properties
             [pscustomobject]@{
+                "Product Name" = $productName
```

Comment:
Some of the product names are quite long.  Can you take a screenshot to show how the table looks now so we can see if it will look ok in practice, at least for commonly used SKUs?

Reply:
<img width="1065" alt="Screenshot 2024-05-07 at 2 13 03 PM" src="https://github.com/cisagov/ScubaGear/assets/147526644/d4532992-94be-44cd-ba81-2038ac9ea7cc">
This is currently how the table looks, and I believe of the 3 tenants, this has the longest product name